subprojects/common/waiter/src/main/kotlin/com/avito/android/waiter/Wait.kt

```diff
@@ -1,19 +1,17 @@
-package com.avito.instrumentation.internal.reservation.adb
+package com.avito.android.waiter
 
 import com.avito.android.Result
 import com.avito.android.Result.Failure
 import com.avito.android.Result.Success
-import com.avito.logger.Logger
-import kotlinx.coroutines.delay
 import java.util.concurrent.TimeUnit
 
-internal suspend fun <T> waitForCondition(
-    logger: Logger,
+public suspend fun <T> waitForCondition(
```

Comment:
it's is now used in two separate modules, and there is no `coroutines` dependency here

Reply:
This action is an example of why `coroutines` are peace of shit
Let's add a TODO to code remove it with FLOW or RX